core/dbt/contracts/graph/model_config.py

```diff
@@ -572,6 +572,7 @@ def same_contents(cls, unrendered: Dict[str, Any], other: Dict[str, Any]) -> boo
             "warn_if",
             "error_if",
             "store_failures",
+            "strategy",
```

Comment:
I think strategy is a little vague, maybe something like: `store_materialization`? That way it's clear that it's related to `store_failures`

Reply:
Agreed. When we first picked `strategy`, I didn't realize how close we were to the generic `Config` object. Maybe `store_failures_as`? Again, I would want input from @fleid on this, and maybe @dataders as well.

Reply:
`store_failures_as` seems pretty intuitive to me